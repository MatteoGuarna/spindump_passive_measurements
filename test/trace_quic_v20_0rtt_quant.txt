A QUIC run with a known 0-RTT usage
