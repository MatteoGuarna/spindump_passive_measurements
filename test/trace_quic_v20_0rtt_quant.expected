[
{ "Event": "new", "Type": "QUIC", "Addrs": ["192.168.203.236","52.58.13.57"], "Session": "af2438c8-8fbb17bf77525bb08e1d45 (58849:4433)", "Ts": 1558426600393329, "State": "Starting", "Packets1": 0, "Packets2": 0, "Bytes1": 0, "Bytes2": 0 },
{ "Event": "change", "Type": "QUIC", "Addrs": ["192.168.203.236","52.58.13.57"], "Session": "af2438c8-5eed824e44f2be50 (58849:4433)", "Ts": 1558426600418093, "State": "Starting", "Packets1": 1, "Packets2": 0, "Bytes1": 1228, "Bytes2": 0, "Bandwidth1": 1228, "Bandwidth2": 0 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["192.168.203.236","52.58.13.57"], "Session": "af2438c8-5eed824e44f2be50 (58849:4433)", "Ts": 1558426600418093, "State": "Up", "Right_rtt": 24764, "Packets1": 1, "Packets2": 0, "Bytes1": 1228, "Bytes2": 0, "Bandwidth1": 1228, "Bandwidth2": 0 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["192.168.203.236","52.58.13.57"], "Session": "af2438c8-5eed824e44f2be50 (58849:4433)", "Ts": 1558426600418099, "State": "Up", "Value": 0, "Who": "responder", "Packets1": 1, "Packets2": 1, "Bytes1": 1228, "Bytes2": 408, "Bandwidth1": 1228, "Bandwidth2": 408 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["192.168.203.236","52.58.13.57"], "Session": "af2438c8-5eed824e44f2be50 (58849:4433)", "Ts": 1558426600418962, "State": "Up", "Value": 0, "Who": "responder", "Packets1": 1, "Packets2": 2, "Bytes1": 1228, "Bytes2": 927, "Bandwidth1": 1228, "Bandwidth2": 927 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["192.168.203.236","52.58.13.57"], "Session": "af2438c8-5eed824e44f2be50 (58849:4433)", "Ts": 1558426600418966, "State": "Up", "Value": 0, "Who": "responder", "Packets1": 1, "Packets2": 3, "Bytes1": 1228, "Bytes2": 2207, "Bandwidth1": 1228, "Bandwidth2": 2207 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["192.168.203.236","52.58.13.57"], "Session": "af2438c8-5eed824e44f2be50 (58849:4433)", "Ts": 1558426600424253, "State": "Up", "Value": 0, "Who": "initiator", "Packets1": 2, "Packets2": 4, "Bytes1": 1372, "Bytes2": 3111, "Bandwidth1": 1372, "Bandwidth2": 3111 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["192.168.203.236","52.58.13.57"], "Session": "af2438c8-5eed824e44f2be50 (58849:4433)", "Ts": 1558426600426613, "State": "Up", "Value": 1, "Who": "initiator", "Packets1": 3, "Packets2": 4, "Bytes1": 1615, "Bytes2": 3111, "Bandwidth1": 1615, "Bandwidth2": 3111 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["192.168.203.236","52.58.13.57"], "Session": "af2438c8-5eed824e44f2be50 (58849:4433)", "Ts": 1558426600427172, "State": "Up", "Value": 1, "Who": "initiator", "Packets1": 4, "Packets2": 4, "Bytes1": 1676, "Bytes2": 3111, "Bandwidth1": 1676, "Bandwidth2": 3111 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["192.168.203.236","52.58.13.57"], "Session": "af2438c8-5eed824e44f2be50 (58849:4433)", "Ts": 1558426600448265, "State": "Up", "Value": 1, "Who": "responder", "Packets1": 5, "Packets2": 5, "Bytes1": 1734, "Bytes2": 3184, "Bandwidth1": 1734, "Bandwidth2": 3184 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["192.168.203.236","52.58.13.57"], "Session": "af2438c8-5eed824e44f2be50 (58849:4433)", "Ts": 1558426600448265, "State": "Up", "Right_rtt": 21652, "Packets1": 5, "Packets2": 5, "Bytes1": 1734, "Bytes2": 3184, "Bandwidth1": 1734, "Bandwidth2": 3184 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["192.168.203.236","52.58.13.57"], "Session": "af2438c8-5eed824e44f2be50 (58849:4433)", "Ts": 1558426600448271, "State": "Up", "Value": 1, "Who": "responder", "Packets1": 5, "Packets2": 6, "Bytes1": 1734, "Bytes2": 3271, "Bandwidth1": 1734, "Bandwidth2": 3271 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["192.168.203.236","52.58.13.57"], "Session": "af2438c8-5eed824e44f2be50 (58849:4433)", "Ts": 1558426600450532, "State": "Up", "Value": 0, "Who": "initiator", "Packets1": 5, "Packets2": 7, "Bytes1": 1734, "Bytes2": 3333, "Bandwidth1": 1734, "Bandwidth2": 3333 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["192.168.203.236","52.58.13.57"], "Session": "af2438c8-5eed824e44f2be50 (58849:4433)", "Ts": 1558426600450532, "State": "Up", "Left_rtt": 2267, "Packets1": 5, "Packets2": 7, "Bytes1": 1734, "Bytes2": 3333, "Bandwidth1": 1734, "Bandwidth2": 3333 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["192.168.203.236","52.58.13.57"], "Session": "af2438c8-5eed824e44f2be50 (58849:4433)", "Ts": 1558426600450532, "State": "Up", "Full_rtt_initiator": 23919, "Packets1": 5, "Packets2": 7, "Bytes1": 1734, "Bytes2": 3333, "Bandwidth1": 1734, "Bandwidth2": 3333 }
]
